lib/shopify_app/webhooks_manager.rb

```diff
@@ -15,6 +18,13 @@ def create_webhooks
       return unless required_webhooks.present?
 
       required_webhooks.each do |webhook|
+        begin
+          is_valid_topic?(webhook[:topic])
+        rescue ShopifyApp::WebhookTopicValidator::InvalidTopic => e
+          Rails.logger.warn "#{@topic} is an invalid webhook topic. "\
+          "Valid topics include: #{VALID_WEBHOOK_TOPICS}"
+          raise
```

Comment:
totally not required but we could probably do something really cool here with a suggestion. Find the closest matching topic using some string difference and Say did you mean ... rather than printing all the topics.

What do you think?